### PR TITLE
[needs testing] ImageMagick: update to 7.0.8.11.

### PR DESCRIPTION
**Pull request by jnbr**
Based on: https://github.com/voidlinux/void-packages/pull/13111

### CODE REVIEW

File: srcpkgs/ImageMagick6/template [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	pkg_install() {
+		vmove "usr/lib/libMagick*.so.*"
+		vmove "usr/lib/ImageMagick*"
+		vmove "usr/share/ImageMagick*"
+	}
+}
+
+libmagick6-devel_package() {
+	depends="
+		fftw-devel fontconfig-devel lcms2-devel libXt-devel
```
Comment (maxice8) [inline srcpkgs/ImageMagick6/template]: this looks ugly IMO

**Comment by chneukirchen** [comment]
For dmtx: https://lab.nexedi.com/nexedi/slapos/commit/23a31a55d3041d79684175fdaa5d3eca3f1db7d8

For zbar: https://bugs.freebsd.org/bugzilla/attachment.cgi?id=185235&action=diff

**Comment by jnbr** [comment]
Thanks for the patches. I added them and moved more packages which didn't link against ImageMagick-7 to ImageMagick-6 (psiconv, inkscape, autotrace).

File: srcpkgs/ImageMagick6/template (hunk 1 above)
Comment (jnbr) [reply srcpkgs/ImageMagick6/template]: is it better now?

**Pull request merged**